src/js/otus/components/Detail/Isolates/IsolateEditor.tsx

```diff
@@ -101,7 +96,7 @@ export default function IsolateEditor() {
         <>
             <SubviewHeader>
                 <IsolateEditorTitle>
-                    Isolates <Badge>{isolateComponents.length}</Badge>
+                    Isolates <Badge className="text-base ml-2">{isolateComponents.length}</Badge>
```

Comment:
Use `flex` and `gap-#` instead  of margin.

Reply:
Used the `ViewHeaderTitleBadge` instead which applies the same styling